README.md

```diff
@@ -231,17 +225,24 @@ repository (i.e. [general issues in Zonemaster](https://github.com/zonemaster/zo
 
 ## Notable bugs and issues
 
-### DNSSEC algorithm 15
+### DNSSEC algorithms 15 and 16
 
 Limitations in the support of DNSSEC algorithm 15 is described above.
 
+### Use of RIPE riswhois for ASN lookup
+
+Test case [Connectivity03] ("AS Diversity") has been updated with the support of using the
+RIPE riswhois service instead of the Cymru service. The Cymru service is the default and
+there is an issue with riswhois service (zonemaster/zonemaster-engine#833). It will be
```

Comment:
Add a link to the issue
```suggestion
there is an issue with riswhois service [zonemaster/zonemaster-engine#833]. It will be

[zonemaster/zonemaster#833]: https://github.com/zonemaster/zonemaster-engine/issues/833
```

Reply:
I thought that Github magically made a link of it, but I forgot to check. Instead of taking your commit (which had an error) I incorporated the change. Thanks.

Reply:
> which had an error

oops yes you're right.